[Docs] Update installation docs (#15156)
pimcore/pimcore: doc/01_Getting_Started/05_Handle_Failed_Jobs.md

# Handle Failed Jobs

> If some maintenance jobs are failed in the processing, they are discarded from the respective transport after defined retries. 

Reviewer: ```suggestion
If maintenance jobs fail during processing, they are discarded from the respective transport after defined number of retries.
```

Reviewer 2: Nice thanks! We restructured the files a bit, but we added your suggestions!
